### DISCUSSION TITLE
Dell S3423DWC issues

### DISCUSSION THREAD

**Discussion by user858753257** [MonitorControl/MonitorControl#1218] [Q&A]
Hi , 

I have seen following behavior:

- brightness control works only in the first 50% 
- audio control doesn't have any effect 

Monitor is directly connected via USB-C with a M1 Book Air . 

DDC Option in Option is on . 

Also the Dell OSX app can't control S-Series displays (on windows it's possible) 

Any hint ?

**Comment by waydabber** [comment]
it might be possible that only that specific input has issues with DDC. Did you try the same input with Windows?
